helm/fluent-logshipping-app/templates/daemonset.yaml

```
@@ -110,17 +111,16 @@ spec:
       - name: var-log
         hostPath:
           path: /var/log
-      - name: var-tmp-fluent-bit
-        hostPath:
-          path: /var/tmp/fluent-bit
-          type: DirectoryOrCreate
       - name: fluent-bit-config
         configMap:
           name: {{ include "resource.default.name" . }}-configmap
       - name: etc-machine-id
         hostPath:
           path: /etc/machine-id
           type: File
+      - name: tmp-storage
+        emptyDir:
+          sizeLimit: {{ .Values.fluentbit.storage.sizeLimit }}
```

```suggestion
          sizeLimit: {{ .Values.fluentbit.storage.sizeLimit }}
```

this is a good idea, are we alerted once the pods crash loop or reach the size?

---

I would Hope so but l created another issue in atlas to have proper monitoring for fluent-bit